app/services/notices/setup/abstraction-alerts/submit-alert-thresholds.service.js

```diff
@@ -38,6 +40,12 @@ async function go(sessionId, payload) {
 }
 
 async function _save(session, payload) {
+  const alertThresholdsKey = 'alert-thresholds'
+
+  session.alertThresholds = Array.isArray(payload[alertThresholdsKey])
+    ? payload[alertThresholdsKey]
+    : [payload[alertThresholdsKey]]
```

Comment:
Personally, if I can't fit a ternary on a single line, then I stick with a standard if/else.

But for this specific scenario, we already have a pattern in place. Check out `app/services/return-versions/setup/submit-points.service.js` and `app/services/return-versions/setup/submit-purpose.service.js`.

The benefit is that it clarifies why the bit of logic exists.

> 😩 I say pattern, I can see they are slightly different 😭 .
>
> I think `submit-points.service.js` is the one to follow (and if you have a mo, please feel free to update `submit-purpose.service.js`